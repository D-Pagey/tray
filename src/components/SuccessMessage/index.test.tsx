import React from 'react';
import { render } from '@testing-library/react';
import { SuccessMessage } from '.';

describe('SuccessMessage component', () => {
    it('should render', () => {
        const { container } = render(<SuccessMessage />);
        expect(container.firstChild).toMatchSnapshot();
    });
});
