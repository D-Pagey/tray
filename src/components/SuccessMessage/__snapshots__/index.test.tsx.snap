// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuccessMessage component should render 1`] = `
<div
  class="sc-AxjAm knSwqB"
  data-testid="successMessage"
>
  <p
    class="sc-AxirZ jiUrUH"
  >
    Please verify your email address, you should have received an email from us already!
  </p>
</div>
`;
